frontend/pendingTranslations.ftl

```diff
@@ -2,6 +2,151 @@
+faq-question-acceptable-use-answer-measure-account = Requiring a { -brand-name-firefox-account(capitalization: "uppercase") } with a verified email address
+faq-question-acceptable-use-answer-measure-unlimited-payment = Requiring payment for a user to create more than five aliases
+faq-question-acceptable-use-answer-measure-rate-limit = Rate-limiting the number of aliases that can be generated in one day
+#   $url (url) - link to the Terms of Service, i.e. https://www.mozilla.org/about/legal/terms/firefox-relay/
+#   $attrs (string) - specific attributes added to external links
+faq-question-acceptable-use-answer-b-html = Please review our <a href="{ $url }" { $attrs }>Terms of Service</a> for more information.
+
+modal-custom-alias-picker-heading = Create a new custom alias
+modal-custom-alias-picker-warning = All you need to do is make up and share a unique alias that uses your custom domain — the alias will be generated automatically. Try “shop@customdomain.mozmail.com” next time you shop online, for example.
+modal-custom-alias-picker-form-heading = Or, create a custom alias manually
+modal-custom-alias-picker-form-prefix-label = Enter alias prefix
+# This is shown in placeholder of the form field in which users can pick a custom alias prefix for their own subdomain,
+# as an example of what email addresses to use (e.g. `coffee@customdomain.mozmail.com`).
+modal-custom-alias-picker-form-prefix-placeholder = e.g. "coffee"
+modal-custom-alias-picker-form-submit-label = Generate Alias
+modal-custom-alias-picker-creation-error = Your custom alias could not be created. Please try again, or send an email to the alias to create it.
+
+popover-custom-alias-explainer-heading = How to create custom aliases
+popover-custom-alias-explainer-explanation = All you need to do is make up and share a unique alias that uses your custom domain — the alias will be generated automatically. Try “shop@customdomain.mozmail.com” next time you shop online, for example.
+popover-custom-alias-explainer-generate-button-heading = Generate a custom alias manually
+popover-custom-alias-explainer-generate-button-label = Generate custom alias
+popover-custom-alias-explainer-close-button-label = Close
+
+# Label for each of the dots representing a tip in a panel in the bottom right-hand corner.
+# Variables
+#   $nr (number) - Which tip can be seen by clicking/tapping this particular dot.
+tips-switcher-label = Tip { $nr }
+
+tips-custom-alias-heading = Creating aliases using your custom domain
+tips-custom-alias-content = All you need to do is make up and share a unique alias that uses your custom domain — the alias will be generated automatically. Try “shop@customdomain.mozmail.com” next time you shop online, for example.
+
```

Comment:
I've got the feeling that there's a bunch of strings in here that aren't actually new in this PR :P Could you remove every string that you didn't introduce?

(Tip: when building the app, it will log the strings that have not been submitted to the l10n repository in the console - the rest can be removed. Look for `The following strings have not yet been merged into the l10n repository, and thus cannot be translated yet:`.)